src/components/MovieDetails.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { Meta, StoryObj } from "@storybook/react";
+import MovieDetails from "./MovieDetails";
+
+
+const meta = {
+    component: MovieDetails,
+    title: 'MovieDetails',
+    tags: ['autodocs'],
+    //👇 Our exports that end in "Data" are not stories.
+    excludeStories: /.*Data$/,
+    args: {},
+} satisfies Meta<typeof MovieDetails>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+const moviesData = {
```

Comment:
Do you repeat the same mock object? Could you create in the separate file and reuse it via export?